packages/expo-dev-menu/ios/DevMenuManager.swift

```diff
@@ -432,4 +432,12 @@ open class DevMenuManager: NSObject {
       CTFontManagerRegisterGraphicsFont(font!, &error)
     }
   }
+  
+  @objc
+  public var registeredCallbacks: [String] = []
+  
+  @objc
+  public func getRegisteredCallbacks() -> [String] {
```

Comment:
do we need a getter method if the property itself is public? (or should we expose a setter method instead of making the property public?)